tests/integrations/aws_lambda/test_aws.py

```diff
@@ -67,7 +67,11 @@ def inner(code, payload):
         # Check file for valid syntax first, and that the integration does not
         # crash when not running in Lambda (but rather a local deployment tool
         # such as chalice's)
-        subprocess.check_call([sys.executable, str(tmp.join("test_lambda.py"))])
+        try:
+            subprocess.check_call([sys.executable, str(tmp.join("test_lambda.py"))])
+        except Exception as e:
+            # Exception caught in case of Initialization error
+            pass
```

Comment:
It doesn't look right to silence the exception here. If calling the subprocess fails for some arbitrary reason, the test should fail.